google/ggrc-core
# GGRC-6746 Do not return objects mapped to Issue through Snapshot as relevant to this Issue

# Issue description

When raising an Issue on Assessment, objects having Snapshot mapped to that Assessment are seen on Issue page, but they should not and only their Snapshots should be present.

# Steps to test the changes

1. Go to Audit page;
2. Create an Assessment and map Snapshot of Control to it;
3. Raise an Issue for the Assessment.
4. Go to Issue page;

Expected result: Control should NOT be present in "Controls" tab on Issue page since it's Snapshot but not Control itself is mapped to this Issue.

5. Go to original Control page;

Expected result: Issue should NOT be present in "Issues" tab on Control page since Control Snapshot but not Control itself is mapped to Issue.

# Solution description

Previously there was special checking for `Issue` object type passed in `relevant` query API operator and in `models.relationship_helper.get_ids_related_to` function which made `relevant` to return not only objects that are mapped to `Issue` directly through `Relationship`, but also objects having `Snapshot` mapped to it.

The solution simply removes this check.

# Sanity checklist




# PR Review checklist
